tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -130,6 +145,11 @@ static int spi_complete_multiple(struct spi_dt_spec *spec)
 		return -1;
 	}
 
+	if (memcmp(tx_bufs[2].buf, rx_bufs[2].buf, BUF3_SIZE)) {
```

Comment:
For consistency with what is done for other buffers:
```suggestion
	if (memcmp(buffer3_tx, buffer3_rx, BUF3_SIZE)) {
```